lua/wikis/valorant/MatchGroup/Input/Custom.lua

```diff
@@ -137,6 +157,160 @@ function MatchFunctions.getPatch(match, games)
 	)
 end
 
+---@return table
+function MatchFunctions.populateOpponentExtradata(match)
+	-- Players are only defined per-map, build the opponent's player list from the maps.
+	Array.forEach(match.opponents, function(opponent, opponentIdx)
+		if not opponent.match2players or #opponent.match2players == 0 then
+			local allPlayers = {}
+			local playerNames = {}
```

Comment:
In what scenario do we need to rebuild the players list? It should be correct no?